linkedevents/settings.py

```diff
@@ -292,6 +300,7 @@ def get_git_revision_hash() -> str:
     "JWT_PAYLOAD_GET_USER_ID_HANDLER": "helusers.jwt.get_user_id_from_payload_handler",
     "JWT_AUDIENCE": env("TOKEN_AUTH_ACCEPTED_AUDIENCE"),
     "JWT_SECRET_KEY": env("TOKEN_AUTH_SHARED_SECRET"),
+    "JWT_AUTH_HEADER_PREFIX": "Bearer",
```

Comment:
I set up tunnistamo locally and tested that without this change the authentication didn't work. :shrug:

Reply:
I did a deep dive into the code and for all I can tell this seems to be the way. But, there's one other Helsinki project with a similar configuration, but without JWT_AUTH_HEADER_PREFIX:

https://github.com/City-of-Helsinki/respa/blob/develop/respa/settings.py#L302 

Hmm hmm..

Reply:
In dev env the backend has the default "JWT" value for  JWT_AUTH_HEADER_PREFIX. However frontend requests depart with "Bearer" (or also seen lowercase "bearer")

Either I'm completely misunderstanding this or there's something that flips Bearer -> JWT in between frontend and backend?

Reply:
@dankiki Any thoughts on this? :sweat_smile:

Reply:
I'm suspicious about using `helusers.jwt.JWTAuthentication` in the first place. Looks like Yrjölä added it into helusers in 2015 ( https://github.com/City-of-Helsinki/django-helusers/commit/469c2932575daaf42d8cec5578c087f4e5c340af ) and around the same time into  linkedevents. helusers readme doesn't mention this at at all - instead helusers tells to use  helusers.oidc.ApiTokenAuthentication for django JWT stuff in the [readme](https://github.com/City-of-Helsinki/django-helusers#django-rest-framework-api-authentication-using-jwt). ApiTokenAuthentication (and RequestJWTAuthentication) are both newer features.

I get a feeling that `helusers.jwt.JWTAuthentication` is unmaintained and exists as mainly because nobody dares to remove it. 

Lets still try to get some confirmation whether something unusual happens to the header on the way though..

Reply:
Since `JWTAuthentication` is the one that is in use currently, I'd say that we could make updating to OIDC a separate task, since that most likely also requires refactoring the settings/pipeline configuration. The logic which I added in this PR should afaik be quite easily transferable over to `helusers.oidc.ApiTokenAuthentication`?

Reply:
My current assumption is that JWTAuthentication is actually overriden by local_settings.py in ninja/prod environments as ApiTokenAuthentication.  The error message I get out with a invalid token from the ninja env matches the error message ApiTokenAuthentication would produce. JWTAuthentication would not produce that message.

But I agree a separate issue might be a good idea to convert all references of JWTAuthentication to ApiTokenAuthentication once we have clarity on this matter.